week2/homework/src/add.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const fs = require('fs');
+
+// Adds a To-do
+var addTodo = title => {
+  var todos = [];
+  var todo = {
+    title,
+  };
+
+  try {
+    var todosString = fs.readFileSync('todos-data.json');
+    todos = JSON.parse(todosString);
```

Comment:
using JSON 👍

Reply:
Hi @gajduk, thanks for the feedback and for your time. The reset command has been added.